src/panels/lovelace/cards/hui-calendar-card.ts

```diff
@@ -222,7 +235,7 @@ export class HuiCalendarCard extends LitElement implements LovelaceCard {
     }
 
     ha-full-calendar {
-      --calendar-height: 400px;
+      height: calc(100% - 16px);
```

Comment:
Did you also test this in a none section view? Does this also work for the masonry view? I think we need to do this conditional like we do for some other cards?

Reply:
Ah yes masonry has issues
<img width="990" height="388" alt="image" src="https://github.com/user-attachments/assets/3712b110-812d-4879-9475-32182f419089" />